Makefile

```diff
@@ -31,7 +38,9 @@ db.migrate.status:
 
 fixtures.generate:
 	@echo "Generating fixtures ..."
-	@docker-compose -f docker-compose-tests.yaml up fixtures-service
+	@if [ -d fixtures ]; then rm -rf fixtures; fi
```

Comment:
why not use variable `FIXTURES_FOLDER` here?

Reply:
good point. Fixed!

Reply:
not completely ... let me push the final fix